src/Provider/GoogleMaps/GoogleMaps.php

```diff
@@ -264,9 +264,42 @@ private function fetchUrl(string $url, string $locale = null, int $limit, string
             if (isset($result->formatted_address)) {
                 $address = $address->withFormattedAddress($result->formatted_address);
             }
+            if ($builder->hasValue('street_address')) {
+                $address = $address->withStreetAddress($builder->getValue('street_address'));
+            }
```

Comment:
Actually, I couldn't find any example that has this parameter in address_component. Also I couldn't find example for `intersection`.

If we remove that `if` for `street_address`, why don't we remove it for all dynamic parameters.. But why should we return bunch of empty parameters?